### PR TITLE
Feature/job detail expanded sql

### PR DESCRIPTION
**Pull request by mell0kat**
expandSQL

### Description

User can expand to see full SQL in a Modal.
User can copy SQL to clipboard.

### Checklist

- [x] This PR contains tests (at least one (non-snapshot) test)

### CODE REVIEW

File: src/components/JobDetailPage.tsx [hunk 1]
```diff
@@ -121,11 +198,11 @@ const JobDetailPage: FunctionComponent<IProps> = props => {
...
...    on mena.uuid = onboarding_response.account_uuid\n        and onboarding_response.rank = 1\n    left outer join spaceman\n        on id.uuid = spaceman.uuid\n    WHERE mena.deleted_at IS NULL\n    GROUP BY\n      id.uuid\n    , id.name\n    , mena.name\n    , mena.legal_name\n    , mena.email\n    , mena.number_of_employees\n    , mena.current_location_id\n    , mena.initial_location_id\n    , mena.founding_date\n    , onboarding_response.date_founded\n    , mena.industry\n    , onboarding_response.indutry\n    , onboarding_response.intended_stay\n    , onboarding_response.primary_use\n    , website_account.facebook_mena\n    , website_account.linkedin_mena\n    , website_account.twitter_mena\n    , website_account.instagram_mena\n    , website_account.other_websites_mena\n    , case when mena.avatar_file_name is not null and trim(mena.avatar_file_name) <> '' then true::boolean\n            else false::boolean\n            end\n    , service_tags.tags\n    , spaceman.legal_name_spaceman\n    , spaceman.short_code_spaceman\n    , spaceman.latest_commitment_location_id_spaceman\n    , spaceman.latest_commitment_reservable_type_spaceman"},"description":"dim_account_member_360_dag"`
```
Comment (grantfosterdev) [inline src/components/JobDetailPage.tsx]: please remove!

**Review by geeeeemoney — CHANGES_REQUESTED** [review]
Looks good, just remove the testing SQL

File: src/components/JobDetailPage.tsx (hunk 1 above)
Comment (mell0kat) [reply src/components/JobDetailPage.tsx]: Good catch!

**Review by geeeeemoney — APPROVED** [review]

File: src/components/JobDetailPage.tsx [hunk 2]
```diff
@@ -83,7 +117,46 @@ const StyledTypographySQL = withStyles({
...
+    backgroundColor: 'white',
+    cursor: 'pointer'
+  }
+})(OpenWithSharpIcon)
+
+const StyledCloseIcon = withStyles({
+  root: {
+    position: 'absolute',
+    right: '2rem',
+    top: '1rem',
```
Comment (jhubley) [inline src/components/JobDetailPage.tsx:135]: A minor thing, but the icon to close the modal is sitting just above the frame of the modal
Screen Shot 2019-12-04 at 1 07 51 PM
Actually it looks like there is a black border at the top of the modal.
Screen Shot 2019-12-04 at 1 03 23 PM
Change line 135 to `    top: '2rem',` to get rid of the overlap?

File: src/components/JobDetailPage.tsx [hunk 3]
```diff
@@ -160,22 +237,39 @@ const JobDetailPage: FunctionComponent<IProps> = props => {
...
+          <Modal aria-labelledby='modal-title' open={SQLModalOpen}>
+            <div className={SQLModalContainer}>
+              {/* Need this extra container for the absolutely-positioned elements */}
+              <StyledCloseIcon fontSize='large' onClick={() => setSQLModalOpen(false)} />(
+              <Typography
+                color='secondary'
+                className={copyToClipboard}
+                onClick={() => {
+                  if (SQL) {
+                    navigator.clipboard.writeText(SQL)
```
Comment (jhubley) [inline src/components/JobDetailPage.tsx:256]: Works well. I think it'll be useful for users to have some acknowledgement that the content has been copied, though, an alert that fades away or something. I'll put a ticket in the backlog

**Review by jhubley — CHANGES_REQUESTED** [review]

File: src/components/JobDetailPage.tsx (hunk 3 above)
Comment (mell0kat) [reply src/components/JobDetailPage.tsx:256]: Yes I agree. Thanks for adding the ticket! @jhubley

**Comment by mell0kat** [comment]
@jhubley Good catch!

So -- weirdly -- the issue was some extra parentheses in the code that was adding an extra line.  Just added a fix (see line 249 of this diff https://github.com/MarquezProject/marquez-web/pull/24/commits/0151d8b110cc24085a01a3e2cc26450f43d4e584#diff-46665bbb0110f32365f8fabb43dc857bL249)

Screen Shot 2019-12-07 at 5 30 03 AM

Going to re-request your review right now

**Review by jhubley — CHANGES_REQUESTED** [review]
Good fix!
The only thing now is that when you scroll down the expanded SQL, seeing the text behind the title looks a little messy:
Screen Shot 2019-12-09 at 12 26 24 PM
Can you change the styling ? This is what I was thinking:
```
SQLModalTitle: {
  right: 10%;
  width: 80%;
  position: fixed;
  font-size: 2rem;
  padding-top: 1rem;
  font-weight: 700;
  background: #fff;
}
```
Sorry to be so picky! :) Thank you!

**Review by jhubley — APPROVED** [review]

**Pull request merged**